front/src/applications/operationalStudies/views/Scenario/components/ImportTimetableItem/ImportTimetableItemConfig.tsx

```diff
@@ -172,6 +178,25 @@ const ImportTimetableItemConfig = ({
     setTrainsJsonData(trainData);
   };
 
+  const processXmlFile = async (file: File, fileContent: string) => {
+    setIsLoading(true);
+
+    if (!railwayManagerUrl) {
+      await locallyProcessXmlFile(fileContent);
```

Comment:
We shoudl setIsLoading(false) before returning
```suggestion
      await locallyProcessXmlFile(fileContent);
      setIsLoading(false)
```